### PR TITLE
Add check for return value of spdm_get_random_number().

### PR DESCRIPTION
**Pull request by qizhangz**
Fix: #356 

Signed-off-by: Qi Zhang <qi1.zhang@intel.com>

### CODE REVIEW

File: library/spdm_responder_lib/libspdm_rsp_measurements.c [hunk 1]
```diff
@@ -83,7 +85,9 @@ void spdm_create_measurement_opaque(IN spdm_context_t *spdm_context,
 	ptr = (void *)((uint8_t *)response_message + response_message_size -
 		       measurment_no_sig_size);
 
-	spdm_get_random_number(SPDM_NONCE_SIZE, ptr);
+	if(!spdm_get_random_number(SPDM_NONCE_SIZE, ptr)) {
```
Comment (steven-bellock) [inline library/spdm_responder_lib/libspdm_rsp_measurements.c:88]: It looks like spdm_create_measurement_opaque needs a return value, but we can do that in a separate pull request.

**Review by steven-bellock — APPROVED** [review]

**Comment by jyao1** [comment]
ref: https://github.com/DMTF/libspdm/issues/360
